legal-api/tests/unit/services/test_document_meta.py

```diff
@@ -101,6 +103,30 @@ def test_available_on_paper_only(session, app):
                     'status': 'COMPLETED',
                     'name': 'changeOfAddress',
                     'availableOnPaperOnly': True,
+                    'availableInColinOnly': False,
+                    'date': FILING_DATE
+                },
+                'business': {
+                    'identifier': 'BC1234567'
+                }
+            }
+        }
+        assert len(document_meta.get_documents(filing)) == 0
+
+
+def test_available_in_colin_only(session, app):
```

Comment:
I didn't understand the purpose of this test as availableInColinOnly doesn't matter for the DocumentMetaService and nothing related to the flag is asserted.

Reply:
it is a test that verifies that no documents are returned when the filing is in colin only